FlipperZero/.swiftlint.yml

```diff
@@ -5,11 +5,81 @@ excluded:
  - Frameworks
 
 disabled_rules:
- - force_try
  - identifier_name
- - line_length
- - force_cast
- - trailing_whitespace
- - opening_brace
+ - todo
+
+opt_in_rules:
+ - anyobject_protocol
+ - array_init
+ - attributes
+ - closure_body_length
+ - closure_end_indentation
+ - closure_spacing
+ - collection_alignment
+ - conditional_returns_on_newline
+ - contains_over_filter_count
+ - contains_over_filter_is_empty
+ - contains_over_first_not_nil
+ - contains_over_range_nil_comparison
+ - convenience_type
+ - discouraged_optional_boolean
+ - discouraged_optional_collection
+ - empty_collection_literal
+ - empty_count
```

Comment:
`empty_count` may be annoying. Consider such an example: `if clients.count > 0` would produce a warning with recommendation to switch to `if !clients.isEmpty`.
I can't insist but comparison to 0 looks more readable to my taste.

Reply:
This is important for performance, because `count` is _O(N)_ operation for sequence, which might be not a collection. Try to run this:
```swift
let a1: [Int] = Array(1...100)
let a2 = a1.lazy.filter({ i in i.isMultiple(of: 3) })
    .compactMap { i -> String in
        print(i)
        return "\(i)"
    }

if a2.isEmpty == false {
    print("is not empty")
}

if a2.count > 0 {
    print("is not empty")
}
```